### PR TITLE
Documentation: invite user outside of the organization

### PR DESCRIPTION
**Pull request by mdacoca**
### Checklist

- [ ] I submit my changes into the `develop` branch
- [ ] I have created a changelog fragment 
- [ ] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues (see [GitHub docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary
  ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
  [cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning),
  [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and
  [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [ ] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.

### CODE REVIEW

File: site/content/en/docs/manual/advanced/organization.md [hunk 1]
```diff
@@ -144,7 +139,46 @@ by going to **Organization page** > **Leave organization**.
...
+3. Click **Ok**.
+4. The person being invited will receive an email with the link.
+
+   ![](/images/invitation_to_org.jpg)
+
+5. Person must click the link and
+   [**set up an account**](/docs/manual/basics/registration/#user-registration).
+
+6. After account is created, the user will be visible
+   on the [**Organization page**](#organization-page)
```
Comment (klakhov) [inline site/content/en/docs/manual/advanced/organization.md:171]: It is a bug, invited user sould be visible immidiately. It is fixed in #7073, but that PR needs more improvents to this feature

**Review by klakhov — COMMENTED** [review]
This PR should wait till improvements of #7073 are implemented and merged, I belive

File: site/content/en/docs/manual/advanced/organization.md (hunk 1 above)
Comment (mdacoca) [reply site/content/en/docs/manual/advanced/organization.md:171]: Ok waiting then

Comment (klakhov) [reply site/content/en/docs/manual/advanced/organization.md:171]: @mdacoca 
Ok, If we want to merge it before, then lets remove `6` from the list. 
Then somewhere in between 4-5 we can add a line about resending/removing invitations. Thats how it looks in UI:
image
You can see with fix from #7073, or admin account on develop.

Basically we can write here that owners/maintainers can resend/remove unaccepted invitations.

**Review by klakhov — APPROVED** [review]
Looks ok, but lets wait for review and merge of #7073. Then merge this PR

**Pull request merged**